src/main/java/net/minecraftforge/fmp/util/IWorldLocation.java

```diff
@@ -0,0 +1,16 @@
+package net.minecraftforge.fmp.util;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Representation of the {@link World} and {@link BlockPos} of an object.
+ */
+public interface IWorldLocation
+{
+    
+    public World getWorldIn();
```

Comment:
This was changed due to obfuscation bugs, but I'm not sure on the details anymore.